README.md

```diff
@@ -284,6 +285,19 @@ func main() {
 }
 ```
 
+## Set Service URL
+To set the service URL,
+
+```go
+service.SetServiceURL("my new url")
+```
+
+Or can set it from external sources. For example, environment variables:
+
+```bash
+export <service name>_url="my new url"
```

Comment:
I think examples of environment variables should use names that are in all caps :)